packages/shacl-language-server/webpack.config.js

```diff
@@ -0,0 +1,162 @@
+const path = require('path');
+const { isCI } = require('ci-info');
+const { BannerPlugin } = require('webpack');
+const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
+const TerserPlugin = require('terser-webpack-plugin');
+
+const SRC_DIR = path.join(__dirname, 'src');
+
+// Don't minify the parser names; this breaks chevrotain. See here: https://sap.github.io/chevrotain/docs/FAQ.html#MINIFIED
+const reserved = [
+  'BaseSparqlParser',
+  'W3SpecSparqlParser',
+  'StardogSparqlParser',
+  'SrsParser',
+  'SmsParser',
+  'TurtleParser',
+  'ShaclParser',
+  'Parser',
+];
+
+const cliConfig = {
+  mode: 'production',
+  target: 'node',
+  entry: path.join(SRC_DIR, 'cli.ts'),
+  output: {
+    path: path.join(__dirname, 'dist'),
+    filename: 'cli.js',
+    library: 'srs-language-server',
```

Comment:
Should this be `shacl-language-server` (and also in the worker config)?

Reply:
Yep, good catch.